core/src/main/resources/schema/20151027185309_schema_changes_for_keystone_v3.sql

```diff
@@ -0,0 +1,30 @@
+-- // schema changes for keystone v3
+-- Migration SQL that makes the change goes here.
+ALTER TABLE credential ADD COLUMN keystoneversion character varying(255);
+
+ALTER TABLE credential ADD COLUMN keystoneauthscope character varying(255);
+
+ALTER TABLE credential ADD COLUMN projectdomainname character varying(255);
+
+ALTER TABLE credential ADD COLUMN domainname character varying(255);
+
+ALTER TABLE credential ADD COLUMN projectname character varying(255);
+
+ALTER TABLE credential ADD COLUMN userdomain character varying(255);
+
```

Comment:
On all the existent openstack credentials the keystoneversion have to be set to `cb-keystone-v2`